Event/AcbFilePreSubmitEvent.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Sherlockode\AdvancedContentBundle\Event;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class AcbFilePreSubmitEvent extends Event
+{
+    public const NAME = 'ACB_FILE.PRE_SUBMIT';
```

Comment:
Lowercase is recommended for the event name: https://symfony.com/doc/current/components/event_dispatcher.html#naming-conventions